src/LoRa.cpp

```diff
@@ -211,6 +212,25 @@ float LoRaClass::packetSnr()
   return ((int8_t)readRegister(REG_PKT_SNR_VALUE)) * 0.25;
 }
 
+double LoRaClass::packetFrequencyError()
```

Comment:
This looks good, any reason to use `double` over `float` here?

Reply:
Or does it even need the fraction part and perhaps should just be a ```long``` like ```setFrequency``` uses?